README.md

```diff
@@ -1,89 +1,494 @@
+is _communication_.
+
+
++ [x] **Asking questions**!
+If you _aren't_ asking questions you aren't learning. It's really _that_ simple.
++ [x] **Writing** - capturing _everything_ you learn.
+Enhancing/improving/editing @dwyl's _extensive_ library
+of learning guides and tutorials.
++ [x] **Sharing** what you are thinking
+_especially_ when you see an area for improvement!
++ [x] ***Informing*** your teammates what you are working on
+by consistently updating the task tracker
+(_e.g: with update comments on a GitHub issue_)
+has multiple benefits which compound over time.
++ [x] **Visual** communication including graphics and ***video***.
+
+### 4. Collaboration
+
+<div align="center">
+    <a href="https://unsplash.com/photos/IgUR1iX0mqM"
+      alt="Pair Programming - Photo credit NESA">
+        <img src="https://user-images.githubusercontent.com/194400/71603902-fc66d180-2b56-11ea-8bac-8486990e9ef3.png" width="800">
+    </a>
+</div>
+
+How to work with other people in a team both in-person and remote.
+This is an incredibly valuable transferrable life skill
+that everyone should master.
+See: https://github.com/dwyl/remote-working
+
+
+### 5. UX/Design Thinking
+
+
+
+You don't have to _label_ yourself as a "designer" or go to college for 4 years
+in order to _think_ like a designer. You just have to _observe_ the world
+and think: "_is this **user-friendly**?_"
+
+![steve-jobs-design-is-how-it-works](https://user-images.githubusercontent.com/194400/72974694-5f026480-3dc7-11ea-9d82-e89165f48d49.jpg)
+
+There are plenty of examples of both good and bad design around us.
+You have the _opportunity_ to improve the world
+and the experience of your fellow humans through design.
+What can be more _exciting_ than that?
+
+Read:
++ https://en.wikipedia.org/wiki/Design_thinking
++ https://en.wikipedia.org/wiki/User-centered_design
++ https://uxplanet.org/user-centered-design-process-and-benefits-fd9e431eb5a9
++ https://99percentinvisible.org/article/norman-doors-dont-know-whether-push-pull-blame-design/
+
+
+### 6. Code
+
+#### “_Everybody should learn to program a computer, <br />because it teaches you how to think._” ~ Steve jobs
+
+We think _everyone_ should learn how to (read/write/understand) code
+because it teaches you to think systematically
+and break down problems into small chunks.
+
+[![image](https://user-images.githubusercontent.com/194400/71601935-a6416080-2b4d-11ea-9ad9-70bd5ada490e.png "Photo by Kevin Ku")](https://unsplash.com/photos/w7ZyuGYNpRQ)
+
+@dwyl we write code to make useful things for real people. <br />
+<!-- We don't write code for hedge funds and high frequency traders
+to [help rich people get richer](https://youtu.be/_HDFegpX5gI). -->
+We do not expect that _all_ the people
+doing the Apprenticeship to write code for their _career_.
+In fact we expect people to gravitate
+to the area of creativity they most enjoy
+and is most fulfilling/valuable to them.
+In the same way that reading, writing and math
```

Comment:
I would suggest replacing this sentence with words to the effect of: But technology and code are at the centre of what we do at dwyl (as well as the world we live in) and we have found that knowing how to code is an _essential_ skill to working effectively here.

Reply:
@iteles please make that as a `suggestion` so that it can _easily_ be merged in.
(_I **agree** with **any** improvement **anyone** can make to this repo..._)

I've been _asking_ people to contribute improvements to this repo for *years* and yet I'm still the only one on the list ... https://github.com/dwyl/apprenticeship/graphs/contributors
![image](https://user-images.githubusercontent.com/194400/73057869-6474b300-3e8a-11ea-8a41-5c7742bd42cf.png)
Not even the people we _paid_ to be apprentices made time to create a PR ... 😞 
I still have faith that _I_ will get better at communicating the _expectations_ and thus future apprentices will _understand_ what they need to do. 🌈

Reply:
Nelson, I _tried_ to make this as a suggestion but it didn't let me. It may have been due to a poor connection or just a temperamental github but I will do so now. You'll notice that I made a number of corrections in a commit, only the bare essential suggestions as comments.